test/runner/index.js

```diff
@@ -84,7 +83,10 @@ program
 
 program.parse(process.argv);
 
-function collectFiles(dir, filters, files) {
+function collectFiles(dir, filters) {
+  if (fs.statSync(dir).isFile())
+    return [dir];
+  const files = [];
   for (const name of fs.readdirSync(dir)) {
     if (fs.lstatSync(path.join(dir, name)).isDirectory()) {
       collectFiles(path.join(dir, name), filters, files);
```

Comment:
This is a recursive call, not sure how this works.

Reply:
it doesn't.